src/main/java/org/broadinstitute/hellbender/utils/Nucleotide.java

```diff
@@ -89,26 +128,25 @@
         Arrays.fill(baseToValue, INVALID);
         Arrays.fill(baseToOrdinal, INVALID.ordinal());
         for (final Nucleotide nucleotide : values) {
+            // Notice that {@code "x & 0xFF"} is needed instead of {@code "(int)x" as
```

Comment:
Actually, can you just use `Byte.toUnsignedInt` everywhere?

It may be worth checking if there are human-readable methods for the other bitwise operations you use throughout, or extracting your own if necessary.